.github/workflows/publish_release.yml

```diff
@@ -0,0 +1,170 @@
+        Get-ChildItem . -Filter LICENCE-DIST.md | Copy-Item -Destination build/package/bin/LICENCE.md -Force -PassThru
+      if: "contains(matrix.os, 'windows')"
+
+    - name: Copy Third Party Artifacts and Licence (macOS)
+      run: |
+          cp -v -f build/thirdparty/*.dylib build/package/lib
+          cp -v -f build/thirdparty/*.json build/package/bin
+          cp -v -f ./LICENCE-DIST.md build/package/lib/LICENCE.md
+      if: "contains(matrix.os, 'macos')"
+
+    - name: Copy Licence (Ubuntu)
+      run: cp -v -f ./LICENCE-DIST.md build/package/lib/LICENCE.md
+      if: "contains(matrix.os, 'ubuntu')"
+
+    - name: Upload Artifacts
+      uses: actions/upload-artifact@v2
+      with:
+        name: novelrt-${{ matrix.os }}-${{ matrix.configuration }}
+        path: build/package
+
+  merge-artifacts:
+    needs: [ build-release, create-release ]
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [ windows-latest, ubuntu-latest, macos-10.15]
+        include:
+        - os: windows-latest
+          name: Windows
+        - os: ubuntu-latest
+          name: Ubuntu
+        - os: macos-10.15
+          name: macOS
+
+    outputs:
+      release-id: ${{ needs.create-release.outputs.id }}
+
+    steps:
+    - uses: actions/download-artifact@v2
+      with:
+        path: artifacts
+    - name: Merge build artifacts
+      run: |
+        mkdir merged
+        for artifact in $(find ./artifacts/ -type d -name "novelrt-${{ matrix.os }}-*"); do
+            cp -RT "$artifact" merged/
+        done
+    - name: Upload merged artifacts
+      uses: actions/upload-artifact@v2
+      with:
+        name: NovelRT-${{ matrix.os }}
+        path: merged
+
+    - uses: actions/download-artifact@v2
+      with:
+        name: NovelRT-${{ matrix.os }}
+
+    - name: Create release artifact
+      run: |
+        cd merged
+        zip -r "../NovelRT-${{ needs.create-release.outputs.tag }}-${{ matrix.name }}.zip" .
+        cd ..
+
+    - name: Add merged artifacts to release
+      run: gh release upload "${{ needs.create-release.outputs.tag }}" "NovelRT-${{ needs.create-release.outputs.tag }}-${{ matrix.name }}.zip" --repo "$GITHUB_REPOSITORY"
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  publish-release:
+    needs: merge-artifacts
+    runs-on: ubuntu-latest
+
+    steps:
+    - run: gh api "repos/$GITHUB_REPOSITORY/releases/${{ needs.merge-artifacts.outputs.release-id }}" -X PATCH -F draft=false --silent
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  delete-release:
+    needs: [ build-release, create-release ]
+    if: ${{ failure() }}
+    runs-on: ubuntu-latest
+    steps:
+    - run: gh release delete "${{ needs.create-release.outputs.tag }}" -y --repo "$GITHUB_REPOSITORY"
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    - run: gh api "repos/$GITHUB_REPOSITORY/git/refs/tags/${{ needs.create-release.outputs.tag}}"
```

Comment:
I don't know how this managed to work in Fabulist but you also need to pass `-X DELETE --silent` to actually delete the ref, otherwise it'll just do a GET request.

Reply:
I don't know how I managed to miss it either - I checked my bash history and it's correct there:
```
$ cat ~/.bash_history | grep "refs/tags"
gh api "repos/novelrt/fabulist/git/refs/tags/v0.2.1" -X DELETE --silent
gh api "repos/novelrt/fabulist/git/refs/tags/v0.2.0" -X DELETE --silent
```
But somehow I just completely missed them from the CI stuff :upside_down_face:

Reply:
_woops_ lol, just fixed this now. Thank you for the heads up!